src/Gulpfile.js

```diff
@@ -69,6 +88,12 @@ function resolveAssetGroupPaths(assetGroup, assetManifestPath) {
     assetGroup.inputPaths = assetGroup.inputs.map(function (inputPath) {
         return path.join(assetGroup.basePath, inputPath);
     });
+    
+    assetGroup.watchPaths = !!assetGroup.watch ?
+        assetGroup.watch.map(function (watchPath) {
+            return path.join(assetGroup.basePath, watchPath);
+        }).concat(assetGroup.inputPaths) : assetGroup.inputPaths;
```

Comment:
I put it in the "resolveAssetGroupPaths, as that is where you were generating the input paths, but I will move it to the watch function.